dashboard/test/controllers/survey_results_controller_test.rb

```diff
@@ -10,13 +10,14 @@ class SurveyResultsControllerTest < ActionController::TestCase
     sign_in @teacher
 
     assert_creates(SurveyResult) do
-      post :create, {survey: {survey2016_ethnicity_asian: 22, survey2016_foodstamps: 3}, format: :json}
+      post :create, {survey: {kind: 'Diversity2016', survey2016_ethnicity_asian: 22, survey2016_foodstamps: 3}, format: :json}
```

Comment:
Oh, I just meant that if you'er posting an additional field from client to server (which I thought is what this test is simulating), to make sure we're whitelisting that on the server side, but I think you are above in the controller change.